client/DotNet6/Armada.Client.Test/Tests.cs

```diff
@@ -0,0 +1,156 @@
+using System;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using RichardSzalay.MockHttp;
+
+namespace GResearch.Armada.Client.Test
+{
+    public class Tests
+    {
+        public static DateTimeOffset now = (DateTimeOffset)DateTime.UtcNow;
+
+        // ARMADA_SERVER should be defined in this process's environment, of the
```

Comment:
I'm trying to test this but I'm unsure what to put here for my local dev armada server.  

Maybe we should have a little README.md that can explain how to set up tests and build?

Reply:
I've tried http://localhost:8080/api in my ARMADA_SERVER env variable but that doesn't seem to work on the local ci..

Reply:
You have to point it at the installed version of armada for testing; the URL isn't here :)

Reply:
Yea but I can create a dev armada server and test against that?  But I don't know what the URL is.  

I don't think we should point this to an isntalled version of Armada for a CI test?  It should be whatever test-e2e-setup creates?